test/powershell/Modules/Microsoft.PowerShell.Utility/Environment-Variables.Tests.ps1

```diff
@@ -42,5 +42,44 @@ Describe "Environment-Variables" -Tags "CI" {
 	$ENV:TESTENVIRONMENTVARIABLE | Should -Not -BeNullOrEmpty
 	$ENV:TESTENVIRONMENTVARIABLE | Should -Be $expected
 
-    }
+	}
+
+	Context "~ in PATH" {
+        AfterEach {
+            $env:PATH = $oldPath
+        }
+
+		BeforeAll {
+            $oldPath = $env:PATH
```

Comment:
I guess it is tabs here and below. Please fix indentations.

Reply:
I noticed the file was mixed formatting, I'll reformat the entire file.